### PR TITLE
S 19 child user profile map

### PR DESCRIPTION
**Pull request by maziar0andriy**
create child model

### CODE REVIEW

File: apps/user_profile/models.py [hunk 1]
```diff
@@ -41,3 +41,50 @@ class Meta:
...
+        ('M', 'male')
+    )
+
+    firstname = models.CharField(
+        max_length=20,
+        blank=False,
+        verbose_name="Введіть ім'я",
+    )
+
+
```
Comment (aodarc) [inline apps/user_profile/models.py]: pls remove blank lines between model fields

File: final_project/settings.py [hunk 2]
```diff
@@ -54,7 +53,8 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [os.path.join(BASE_DIR, 'templates')],
+        'DIRS': [os.path.join(BASE_DIR, 'templates')]
+        ,
```
Comment (aodarc) [inline final_project/settings.py]: why is it there)?

File: final_project/settings.py [hunk 3]
```diff
@@ -118,8 +118,9 @@
 MEDIA_URL = '/media/'
 
 STATIC_URL = '/static/'
-STATIC_ROOT = 'dist'
+STATIC_ROOT = 'static'
```
Comment (aodarc) [inline final_project/settings.py]: revert this

File: templates/base.html [hunk 4]
```diff
@@ -1,30 +1,17 @@
...
-  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-  <meta name="description" content="">
-  <meta name="author" content="">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link rel="icon" href="../../../../favicon.ico">
 
-  {% block header %}
```
Comment (aodarc) [inline templates/base.html]: why was it deleted?

File: templates/base.html [hunk 5]
```diff
@@ -1,30 +1,17 @@
...
-</head>
-
-<body>
-
-{% block main %}
-{% endblock %}
-
-<script src="{% static 'final_project/jquery/jquery.slim.min.js' %}"></script>
-<script src="{% static 'final_project/popper/popper.min.js' %}"></script>
-<script src="{% static 'final_project/bootstrap/js/bootstrap.min.js' %}"></script>
```
Comment (aodarc) [inline templates/base.html]: revert html files

**Review by aodarc — CHANGES_REQUESTED** [review]
follow my comments

File: final_project/settings.py (hunk 2 above)
Comment (maziar0andriy) [reply final_project/settings.py]: because it was here))) delete it?

File: final_project/settings.py (hunk 3 above)
Comment (maziar0andriy) [reply final_project/settings.py]: ok

File: templates/base.html (hunk 4 above)
Comment (maziar0andriy) [reply templates/base.html]: it so happened))

File: templates/base.html (hunk 5 above)
Comment (maziar0andriy) [reply templates/base.html]: ok

File: apps/user_profile/models.py (hunk 1 above)
Comment (maziar0andriy) [reply apps/user_profile/models.py]: ok

**Pull request merged**